com.avaloq.tools.ddk.xtext.builder/src/com/avaloq/tools/ddk/xtext/builder/MonitoredClusteringBuilderState.java

```diff
@@ -749,24 +751,6 @@ protected void awaitBinaryStorageExecutorQuiescence() {
     }
   }
 
-  /**
-   * Log the first and last 10 StackOverflowError Stack Trace.
-   *
-   * @param ex
-   *          the StackOverflowError
-   * @param binding
-   *          the object to be inserted into the message
-   */
-  protected void logStackOverflowErrorStackTrace(final StackOverflowError ex, final URI binding) {

```

Comment:
since this is a FP, to keep changes to the minimum, I would not delete this method, so you that you do not need to adapt subclasses.

Reply:
Good idea, will keep the method. Also need to modify the PR to be against v9.4.x, I think.